app/models/organization.rb

```diff
@@ -71,12 +71,5 @@ def self.create_root_ns(session_id:)
 
   def self.create_defaults(session_id:)
     create_root_ns(session_id: session_id)
-    organizations = []
-    organizations << { name: "rc", title: "Research Computing" }
-    organizations << { name: "pppl", title: "Princeton Physics Plasma Lab" }
-    organizations << { name: "pul", title: "Princeton University Library" }
-    organizations.each do |org|
-      Organization.create!(org[:name], org[:title], session_id: session_id)
-    end
```

Comment:
We are not using these anymore so I am preventing the code from creating them. I created a separate ticket to do a more in depth review of this class: https://github.com/pulibrary/tiger-data-app/issues/228